tests/core/test_accounts.py

```diff
@@ -97,6 +99,22 @@ def acct(request):
     return Account
 
 
+def test_eth_account_default_kdf():
+    assert eth_account.account.defaultKDF == 'scrypt'
+
+
+def test_eth_account_default_kdf_from_env():
+    import importlib
+    os.environ['ETH_ACCOUNT_KDF'] = 'pbkdf2'
+    importlib.reload(eth_account.account)
+    assert eth_account.account.defaultKDF == 'pbkdf2'
+
+    # This is done to ensure we use the default (scrypt)
+    # KDF for subsequent tests
```

Comment:
Need to dig into this... I'm confused why this doesn't break subsequent tests that don't provide a KDF to `#encrypt` as I would assume the default for those would be `pdkdf2` unless (a) this module reloading / env updating is locally scoped or (b) the `_defaultKDF = os.getenv('ETH_ACCOUNT_KDF' , 'scrypt')` is lazily evaluated.

Reply:
Yeah, that is odd and yes, I would expect tests to break.  You might throw a debugger statement into another test to see why it isn't.

Reply:
Still need to dig into this to make sure we're still not in that weird state, but the test structure has changed:
https://github.com/ethereum/eth-account/pull/38/commits/8aec325d1b02c045143b0106e6a807524c04c6b6#diff-1de692d823115945c2232e1aa3066811R101